README.md

```diff
@@ -1,37 +1,134 @@
-## Prerequisites
-* [We used Jammy ubuntu VM ](https://github.com/Alliedium/awesome-proxmox)
-* [Ansible is installed](https://github.com/Alliedium/awesome-ansible/)
+## This ansible playbook installs the [`Jenkins`](https://www.jenkins.io/doc/) on the specified host ##
```

Comment:
Remove the, articles (the, a, an) are not used in titles (this remark was already in the previous review)